components/common/src/ui.rs

```diff
@@ -74,10 +75,11 @@ impl Status {
             Status::Deleting => ('☒', "Deleting".into(), Colour::Green),
             Status::Demoted => ('✓', "Demoted".into(), Colour::Green),
             Status::Demoting => ('→', "Demoting".into(), Colour::Green),
-            Status::Determining => ('→', "Determining".into(), Colour::Green),
+            Status::Determining => ('☁', "Determining".into(), Colour::Green),
```

Comment:
I love the ☁️ as the glyph for determining, but I wonder if it might be misleading since we use `Status::Determining` in some offline contexts.

Reply:
I looked at where this type is used and it looks like it was added for Builder job status subcommands. I was thinking of less an internet cloud, but rather a thought bubble representing thinking about something 😄 

One takeaway when I changed this was to notice that some of the other types of iconography could be better represented (like promoted (up?), demoted (down?)). I might look at that afterwards as a separate change (it's on my whiteboard right now and doesn't hurt anything in the meantime).

Reply:
That sounds like a good approach. If we want this to be a thought balloon, there is an emoji for that: 💭. That sounds like a good change to make.

Or if we really want to go all-in: 🤔

👍 on a separate change to clean these up more generally. It may seem like a trivial thing, but I feel it has an impact un usability.